### DISCUSSION TITLE
How to show full length of edge label?..

### DISCUSSION THREAD

**Discussion by SmirnovArtyom516** [reaviz/reagraph#125] [Q&A]
Hello everyone, i have a question, i have node and relations between them, also i have  a size of edges which depends on the total percentage of occurrences in node, i want to showing this procentage on edge labels, but when I enter a percentage in the label it is replaced by 3 dots, so maybe i can regular this by myself? Thank you in advance. 

P.S. 
I also have a question about edges, can i somehow regular length of edges, to make it shorter for example?

**Comment by amcdnl** [comment]
This is interesting - Would you mind PR'ing a demo storybook story for this and i can take a look?

**Reply by SmirnovArtyom516** [reply]
i dont really know how to do that, u mean like codesandbox example?

**Reply by amcdnl** [reply]
sure

**Reply by SmirnovArtyom516** [reply]
https://codesandbox.io/s/peaceful-cannon-j4l8zm
